.github/workflows/publish-release.yml

```
@@ -15,7 +15,7 @@ name: publish-release
         type: string
       node-version:
         description: The Nodejs version to use
-        default: 12
+        default: 16
```

Revert in favor of https://github.com/mdn/workflows/pull/118:

```suggestion
        default: 12
```